lib/Backends/NNPI/tests/NNPIParameterSweepTest.cpp

```diff
@@ -18,3 +18,137 @@
+            "FCTest_Float16/30",          "FCTest_Float16/31",
+            "FCTest_Float16/32",          "FCTest_Float16/33",
+            "FCTest_Float16/34",          "FCTest_Float16/47",
+            "FCTest_Float16/48",          "FCTest_Float16/49",
+            "FCTest_Float16/51",          "FCTest_Float16/52",
+            "FCTest_Float16/53",          "FCTest_Float16/54",
+            "FCTest_Float16/55",          "FCTest_Float16/56",
+            "FCTest_Float16/57",          "FCTest_Float16/58",
+            "FCTest_Float16/59",          "FCTest_Float16/60",
+            "FCTest_Float16/61",          "FCTest_Float16/62",
+            "FCTest_Float16/63",          "FCTest_Float16/64",
+            "FCTest_Float16/65",          "FCTest_Float16/66",
+            "FCTest_Float16/67",          "FCTest_Float16/68",
+            "FCTest_Float16/69",          "FCTest_Float16/83",
+            "FCTest_Float16/84",          "FCTest_Float16/86",
+            "FCTest_Float16/87",          "FCTest_Float16/88",
+            "FCTest_Float16/89",          "FCTest_Float16/90",
+            "FCTest_Float16/91",          "FCTest_Float16/92",
+            "FCTest_Float16/93",          "FCTest_Float16/94",
+            "FCTest_Float16/95",          "FCTest_Float16/96",
+            "FCTest_Float16/97",          "FCTest_Float16/98",
+            "FCTest_Float16/99",          "FCTest_Float16/100",
+            "FCTest_Float16/101",         "FCTest_Float16/102",
+            "FCTest_Float16/103",         "FCTest_Float16/104",
+            "FCTest_Float16/116",         "FCTest_Float16/117",
+            "FCTest_Float16/118",         "FCTest_Float16/119",
+            "FCTest_Float16/121",         "FCTest_Float16/122",
+            "FCTest_Float16/123",         "FCTest_Float16/124",
+            "FCTest_Float16/125",         "FCTest_Float16/126",
+            "FCTest_Float16/127",         "FCTest_Float16/128",
+            "FCTest_Float16/129",         "FCTest_Float16/130",
+            "FCTest_Float16/131",         "FCTest_Float16/132",
+            "FCTest_Float16/133",         "FCTest_Float16/134",
+            "FCTest_Float16/135",         "FCTest_Float16/136",
+            "FCTest_Float16/137",         "FCTest_Float16/138",
+            "FCTest_Float16/139",         "FCTest_Int8/139",
+        };
+        for (int i = 0; i < 80; i++) {
+          backendTestBlacklist.insert("RWQSLWS_Float16/" + std::to_string(i));
+        }
+        for (int i = 0; i < 80; i++) {
+          backendTestBlacklist.insert("FRWQSLWS_Float16/" + std::to_string(i));
+        }
+        for (int i = 0; i < 80; i++) {
+          backendTestBlacklist.insert("RWQSLWS_Float16_AccumFloat16/" +
+                                      std::to_string(i));
+        }
+        for (int i = 0; i < 80; i++) {
+          backendTestBlacklist.insert("FRWQSLWS_Float16_AccumFloat16/" +
+                                      std::to_string(i));
+        }
+      }
+    }
+  }
+} emuTests;
```

Comment:
This change is related to https://github.com/pytorch/glow/pull/3976
If that's an issue you can ignore the changes in this file.

Reply:
Going to get #3976 landed separately (it has new changes not included here anyway).